pylint/message/message_definition_store.py

```diff
@@ -21,17 +21,20 @@
 class MessageDefinitionStore:
 
     """The messages store knows information about every possible message definition but
-    has no particular state during analysis.
+    has no particular state during analysis other than py_version.
```

Comment:
I'm not convinced of the need to add this, actually.

Reply:
Hmm, yeah the intent of the design might be broken here, maybe we do need to give the py_version all the way to list the enabled message ? On the other hand, what's the worst that could happen? Having py_version here is convenient.

Reply:
`py_version` is not really a state that changes during analysis, so next time I push I'll just revert this docs change.

Reply:
👍

Reply:
```suggestion
    has no particular state during analysis.
```